portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+/** Class containing server statistics. */
+public final class Comment {
+  String name;
+  String message;
+  String timeCreated;
+  Long id;
+
+  public Comment(String name, String message,  String timestamp, Long id) {
+    this.name = name;
+    this.message = message;    
+    this.timeCreated = timestamp;
```

Comment:
I would name it createDate or similar. timstamp usually means the epoch time. usually, for website, server saves epoch time when the request commited and then client convert epoch time to whatever format they want.

Reply:
Ahh ok, I will create epoch timestamp for the server and, as you mentioned, format on the client, thank you!